husky_components/column-navigation/main.js

```diff
@@ -1339,7 +1339,9 @@ define(function() {
          */
         mark: function(id, item) {
             var $element = this.$find('li[data-id="' + id + '"]');
-            if (!!$element.length) {
+
+            // mark only non-ghost pages if showActionButtonOnGhost is disabled
+            if (!!$element.length && (item.type.name !== 'ghost' || this.options.showActionButtonOnGhost)) {
```

Comment:
I get an error on this line, when double clicking on an element I should be able to add:

```
husky.js:42880 Uncaught TypeError: Cannot read property 'name' of undefined
```